src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -2959,7 +2959,7 @@ MPRadosSerializer::MPRadosSerializer(const DoutPrefixProvider *dpp, RadosStore*
   oid = raw_obj.oid;
   store->getRados()->get_obj_data_pool(obj->get_bucket()->get_placement_rule(),
 				       obj->get_obj(), &meta_pool);
-  store->getRados()->open_pool_ctx(dpp, meta_pool, ioctx, true);
+  store->getRados()->open_pool_ctx(dpp, meta_pool, ioctx, true, false);
```

Comment:
`meta_pool` came from `get_obj_data_pool()`, so i think we want bulk=true here

Reply:
ack, is it ok that this meta pool will be both `mostly_omap` & `bulk` flagged?